about.html

```diff
@@ -0,0 +1,141 @@
+
+          <div class="mobileMenu"> 
+            <div class="m1"><h2 class="m2">Winners Chapel</h2></div>
+            <span class="closeMenu">&times;</span>
+              <ul class="container">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="main.html">Home</a></li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>News</li>
+             </ul>
+            </div>
+
+        </article>
+    <main class="pg1">
+       <a href=""><img src="./images/about-bg.png" alt="" class="bckg1"></a>
+            <h2 class="theme">
+                "Youth For Christ"
+             </h2>
+             <h1 class="ch-name">Winners Chapel Youth Conference 2022.</h1>
+            <div class="moment">
+                 <p class="prg">the conference is to bring together all youth in all church in the nation under one canmopy to worship God.</p>
+            </div>
+            <div class="contact">
+                <p>please contact us per email for any further questions about winners chapel youth conference! </p>
+                <a  href="">winnerconf@gmail.com</a>
+            </div>
+        </main>
+    
+        <div>
+           <div class="logo1">
+
+                <h3> winners chapel youth conference 2022 logo</h3>
+                <p> winners chapel youth conference 2022 logo ws decided by the youth president bhimself alongside some executives</p>
+            
+            <div class="logo-p">
+                <a href=""><img src="./images/download.jpg" alt="" class="logo-ab"></a>
+            </div>
+        </div>
+        <hr class="line"> 
+    </div>
+<section>
+        <h3 class="abh2">See The Past National Youth Conference</h3>
+       <hr class="hr-f"> 
+        <p class="abp">Take a look at the past two conference which took place in abuja, nigeria</p>
+  <section class="di">    
+<div class="card">
+    <div class="image">
+        <div class="bckg">
+        </div>   
+    </div>
+    <div class="text" >
+    <p class="da">2021 </p>
+<p> youth national conference in lagos</p>
+</div>
+</div>
+
+<div class="card">
+    <div class="image1">
+        <div class="bckg">
+        </div>   
+    </div>
+    <div class="text" >
+    <p class="da">2020 </p>
+<p> youth national conference in Abuja</p>
+</div>
+</div>
+</section>  
+ 
+</section>
+  <FOoter class="footer">
+      
+<h2 class="f-headrer">Partner</h2>
+<hr class="hr-f">
+<div class="grid">
+<a href=""><img src="./images/airbnb.png" alt="" class="footer-img "></a>
+<a href=""><img src="./images/mozilla.png" alt="" class="footer-img "></a>
+<a href=""><img src="./images/naver.png" alt="" class="footer-img "></a>
+<a href=""><img src="./images/google.png" alt="" class="footer-img "></a>
+</div>  
+<div class="ff1">
+    <a href=""><img src="./images/download-removebg-preview.png" alt=" " class="f-img1" ></a>
+    <p class="f-p">2022 nation youth conference</p>
+</div>
+  </FOoter>
```

Comment:
- [ ] On the `main` page I can't see `footer` on the mobile size please add that and remember it should have a white background

Reply:
please ma, from the design for the mobile, its only the about me page that has a footer which i included

Reply:
Hello @Richez003, I just checked yes the design doesn't have `footer` take it as optional